src/main/res/layout/loading_image_view.xml

```diff
@@ -7,7 +7,8 @@
         android:id="@+id/imageView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_gravity="center"/>
+        android:layout_gravity="center"
+        android:contentDescription="@string/loading_image_view_description"/>
```

Comment:
i *think* if you `@null` the `contentDescription` in xml, then whoever sets the image can also set the content description in java...


... if you wanted to be slick, you could force users of LoadingImageView to pass a contentDescription by updating these two setters to take in either a string or resID:

```
    public void setImageResource(int resourceId) {
        imageView.setImageResource(resourceId);
    }

    public void setImageDrawable(Drawable drawable) {
        imageView.setImageDrawable(drawable);
    }
```